conf/messages

```diff
@@ -162,6 +162,25 @@ technicalIssues.end.error.invalid = The date the technology issues ended must be
 technicalIssues.end.error.notInFuture = The date the technology issues ended must be today or in the past
 technicalIssues.end.error.endDateLessThanStartDate = The date the technology issues ended must be the same or after they began
 
+# Was hospital stay required page
+# ----------------------------------------------------------
+healthReason.wasHospitalStayRequired.title = Did this health issue include an unexpected hospital stay?
+healthReason.wasHospitalStayRequired.heading = Did this health issue include an unexpected hospital stay?
+healthReason.wasHospitalStayRequired.yes = Yes
+healthReason.wasHospitalStayRequired.no = No
+healthReason.wasHospitalStayRequired.error.required = Tell us if the health issue included a hospital stay
```

Comment:
This needs to be `Tell us if you or someone else was admitted to hospital unexpectedly`